@kiva/kv-components/vue/KvLightbox.vue

```diff
@@ -218,10 +218,16 @@ export default {
 		const kvLightboxBody = ref(null);
 		const controlsRef = ref(null);
 
+		const trapElements = computed(() => [
+			kvLightbox.value, // This lightbox
+			'[role="alert"]', // Any open toasts/alerts on the page
+		]);
 		const {
 			activate: activateFocusTrap,
 			deactivate: deactivateFocusTrap,
-		} = useFocusTrap(kvLightbox);
+		} = useFocusTrap(trapElements, {
+			allowOutsideClick: true, // allow clicking outside the lightbox to close it
```

Comment:
Does this need to be conditional based on props sent to the lightbox? We don't allow outside clicks in certain circumstances such as employee verification for campaign pages.

Reply:
This won't bypass our existing `preventClose` logic. If preventClose is true, clicking on the background won't close the lightbox. This just allows clicking on the background to close the lightbox when preventClose is false.